src/walletConnect/screens/Logos.tsx

```diff
@@ -16,11 +14,18 @@ interface Props {
 }
 
 export default function Logos({ dappImageUrl, dappName }: Props) {
-  const dappConnectInfo = useSelector(dappConnectInfoSelector)
-  if (!dappImageUrl && dappConnectInfo !== DappConnectInfo.Basic) {
+  if (!dappImageUrl) {
     return null
   }
 
+  let isDappImageSvg = false
+  try {
+    const parsedUrl = new URL(dappImageUrl ?? '')
+    isDappImageSvg = parsedUrl.pathname.endsWith('.svg')
```

Comment:
```suggestion
    isDappImageSvg = parsedUrl.pathname.toLowerCase().endsWith('.svg')
```

I wonder if we should useMemo to avoid parsing it on each render 🤔

Reply:
i didn't bother with memo-ing here because it is such a small component that exists for a short time? the computation also seems quite inexpensive?